waiter/src/waiter/metrics.clj

```diff
@@ -449,6 +449,16 @@
      (~handle-elapsed-nanos-fn elapsed#)
      out#))
 
+(defmacro with-timer
+  "Times the operation specified by body using the provided
+  timer and calls handle-elapsed-nanos-fn with the epased
+  time in nanoseconds. Returns [value of body, nanos]."
```

Comment:
Returns a map with keys `:out` and `:elapsed`.

Reply:
Fixed